packages/itwin/grouping-mapping-widget/package.json

```diff
@@ -0,0 +1,116 @@
+{
+  "name": "@itwin/grouping-mapping-widget",
+  "version": "0.1.0",
+  "description": "An iTwin.js 3D Viewer Widget that interfaces with the iTwin Reporting Platform.",
+  "keywords": [
+    "Bentley",
+    "BIM",
+    "iModel",
+    "Reporting"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/imodeljs/viewer-components-react/tree/master/packages/itwin/grouping-mapping-widget"
+  },
+  "license": "MIT",
+  "author": {
+    "name": "Bentley Systems, Inc.",
+    "url": "http://www.bentley.com"
+  },
+  "main": "lib/cjs/grouping-mapping-widget.js",
+  "module": "lib/esm/grouping-mapping-widget.js",
+  "typings": "lib/cjs/grouping-mapping-widget",
+  "scripts": {
+    "build": "npm run -s dual-build && npm run -s copy:assets",
+    "build:cjs": "tsc 1>&2 --outDir lib/cjs",
+    "build:esm": "tsc 1>&2 --module ES2020 --outDir lib/esm",
+    "clean": "rimraf lib",
+    "copy:assets": "npm run -s copy:public && npm run -s copy:cjs && npm run -s copy:esm",
+    "copy:cjs": "cpx \"./src/**/*.scss\" ./lib/cjs",
+    "copy:esm": "cpx \"./src/**/*.scss\" ./lib/esm",
+    "copy:public": "cpx \"./public/**/*\" ./lib/public",
+    "cover": "",
+    "dual-build": "npm run -s build:cjs && npm run -s build:esm",
+    "lint": "eslint -f visualstudio \"./src/**/*.{ts,tsx}\" 1>&2",
+    "lint:fix": "npm run lint -- --fix",
+    "pseudolocalize": "betools pseudolocalize --englishDir ./public/locales/en --out ./public/locales/en-PSEUDO",
+    "rebuild": "npm run clean && npm run build",
+    "test": ""
+  },
+  "dependencies": {
+    "@itwin/imodel-browser-react": "^0.12.1",
```

Comment:
Do you need this dependency? I wouldn't expect iModel browsing to happen within the widget itself.

Reply:
I thought the same thing, but they do use it :/